src/yunohost/utils/yunopaste.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+import requests
+import json
+import errno
+
+from moulinette.core import MoulinetteError
+
+def yunopaste(data):
+
+    paste_server = "https://paste.yunohost.org"
+
+    try:
+        r = requests.post("%s/documents" % paste_server, data=data)
```

Comment:
No timeout set, we should ALWAYS have a timeout on requests calls to avoid freezing the whole YunoHost.